src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,46 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+    private static final int EMPTY_STACK_VALUE = -1;
+    private static final int STACK_CAPACITY = 12;
+    //wszystkie atrybuty podlegają hermetyzacji
+
+    private int[] numbers = new int[STACK_CAPACITY];
+
+    private int total = EMPTY_STACK_VALUE;
+
+    public void countIn(int in) {
+        if (!isFull())
+            numbers[++total] = in;
+    }
+
+    public boolean callCheck() {
+        return total == EMPTY_STACK_VALUE;
```

Comment:
Errors in collaborators work (lack of good review). Extracting constants is done incorrectly. Have you used review hints ('Github - PR checkpoint' at wikamp).
This constant should be different (separate) than the one in peekaboo and countOut methods.